packages/realm-react/CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contribution Guide
+## Introduction
+We are extremely happy for any contributions to `@realm/react`.  Realm is a very busy team and has many things to work on.  If you are having an issue with this package and want to take it on yourself, feel free to open a PR and we will help guide you to get it merged.
```

Comment:
Aren't we all busy? It slightly hints that a contributor might not be. I would probably leave that out.

```suggestion
We are extremely happy for any contributions to `@realm/react`.  While we (the Realm team) are working hard to provide you the best possible experience as developers, we have a long backlog of things we want to do. If you are having an issue with or an idea for this package and want to take it on yourself, feel free to open a PR and we will help guide you to get it merged.
```